### PR TITLE
[JENKINS-60406] Replace RUN_SCRIPTS with ADMINISTER

### PR DESCRIPTION
**Pull request by bitwiseman**
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by car-roll — APPROVED** [review]

File: workflow-cps-global-lib.iml [hunk 1]
```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jglick) [inline workflow-cps-global-lib.iml:1]: Can you ignore this please?

**Review by jglick — CHANGES_REQUESTED** [review]
Forgot https://github.com/jenkinsci/workflow-cps-global-lib-plugin/blob/a1de00ea889a91e3cb4d143c0ba5efb6a7815db3/src/main/resources/org/jenkinsci/plugins/workflow/libs/GlobalLibraries/config.jelly#L28

**Review by jtnord — APPROVED** [review]
Looks good to me.

It may be the code can be further cleaned up as global configuration should never be called unless you have administer, but that will open a can of worms so this looks like a reasonable step that can be potentially refined at some point in the future.

**Review by olamy — APPROVED** [review]
LGTM (no much more to say than @jglick comments)

**Comment by bitwiseman** [comment]
@jglick Addressed feedback.

**Comment by car-roll** [comment]
looks like @jglick 's comment has been addressed, going ahead to merge

**Pull request merged**